docs/workspace/ipython/shell.ipynb

```diff
@@ -11,7 +11,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The IPython Notebook allows a whole range of UNIX/Linux commands. These are presented below as an overview. To begin with, the command `ls` lists the files in the current working directory:"
+    "The IPython Notebook allows simple UNIX/Linux commands to be
+executed in a single input cell. There are no limits but when using,
+please keep in mind that in contrast to a regular UNIX/Linux shell,
+start each shell command with a `!`, for example `!ls` for the command
+`ls` (see below for further explanations about the command).
+Furthermore, each shell command is executed in its own subshell. For
```

Comment:
This is described in more detail in workspace/ipython/unix-shell/file-system.ipynb. Possibly the notebook shell.ipynb should be deleted and the contents integrated into workspace/ipython/unix-shell/.